eventlet/timeout.py

```diff
@@ -175,5 +175,8 @@ def fun(*args, **kwargs):
 
 
 def is_timeout(obj):
-    py3err = getattr(__builtins__, 'TimeoutError', Timeout)
+    if isinstance(__builtins__, dict):  # seen when running tests on py310, but HOW??
```

Comment:
Would be great to check it once outside function body to improve performance.

Reply:
Done.